infrastructure/src/main/java/org/corfudb/infrastructure/log/Compression/ZSTDCompression.java

```diff
@@ -0,0 +1,67 @@
+package org.corfudb.infrastructure.log.Compression;
+
+import com.github.luben.zstd.Zstd;
+
+import java.nio.ByteBuffer;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class ZSTDCompression implements Codec {
+
+    private static final int DEFAULT_COMPRESSION_LEVEL = 3;
```

Comment:
can you make this parameter non-static? When a parameter is static it gets not configurable, it is hardcoded

Reply:
I can make it configurable, but it can't be used unless its piped all the way through the server configs, which I think is very messy. I will create a different patch to try to clean up the Config part. Based on a cursory survey, level 3 seems like a sane value.